envoimoinscher.php

```diff
@@ -2811,7 +3041,7 @@ public function makeApiCall($params)
 		$offers_quotation = array();
 		if (count($offers_db) == 1) // correction needed for quotation
 		{
-			$cot_cl->getQuotation($quot_info);
+			@$cot_cl->getQuotation($quot_info);
```

Comment:
Okay, I made a mistake between cURL and SOAP calls.

Anyway, you should add to have a better management of errors:
```php
	$result = curl_exec($req);
	// Check if something went wrong
	if ($result === false)
		[...]
```